src/components/Article/Article.jsx

```diff
@@ -1,6 +1,67 @@
-// import the React library here
-// import a css file containig article styles
 
-// Create an Article function returning the HTML of article block
+import React from 'react';
+import './Article.css';
 
-// Add a default export statement for Article component to use it in the other files
+const Article = () => (
```

Comment:
```suggestion
export default function Article = () => (
```
you can export it right here

Reply:
I replesed this expression here, but i don't do it in another components